### PR TITLE
Remove sd

### PR DESCRIPTION
**Pull request by shaikatz**

### CODE REVIEW

File: pkg/controller/kubernetes-icinga.go [hunk 1]
```diff
@@ -90,23 +90,8 @@ func main() {
 		DefaultVars:  defaultVars,
 	}
 
-	switch os.Getenv("MAPPING") {
```
Comment (AmitBenAmi) [inline pkg/controller/kubernetes-icinga.go:93]: Did you remove it also from the ConfigMap? at ```deploy/configmap.crd```

**Comment by AmitBenAmi** [comment]
You should also delete the RBAC of all kubernetes resources not needed anymore (pods, namespaces, etc...)

File: go.mod [hunk 2]
```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/Nexinto/go-icinga2-client v0.0.0-20180829072643-d4f6001a2110
+	github.com/iljaweis/kube-controller-gen v0.1.0 // indirect
```
Comment (AmitBenAmi) [inline go.mod:7]: How come you added this?

Comment (shaikatz) [reply go.mod]: Found it in one of the upstream issues, this is how we regenerate the controller (with the YAML file).

File: pkg/controller/kubernetes-icinga.go (hunk 1 above)
Comment (shaikatz) [reply pkg/controller/kubernetes-icinga.go:93]: Will do.

**Comment by shaikatz** [comment]
Fixed the comments.

File: pkg/controller/controller-gen.yaml [hunk 3]
```diff
@@ -1,63 +1,13 @@
 package: main
 imports: |
   "github.com/Nexinto/go-icinga2-client/icinga2"
+  "k8s.io/client-go/kubernetes"
 controllerextra: |
+  Kubernetes kubernetes.Interface
```
Comment (AmitBenAmi) [inline pkg/controller/controller-gen.yaml:6]: What is it used for?

File: deploy/rbac.yaml [hunk 4]
```diff
@@ -12,18 +12,6 @@ metadata:
...
-  - namespaces
-  - pods
-  - replicasets
-  - daemonsets
-  - deployments
-  - statefulsets
-  - services
-  - nodes
-  - componentstatuses
   verbs:
```
Comment (AmitBenAmi) [inline deploy/rbac.yaml:15]: Can you delete the verbs also?

File: pkg/controller/controller-gen.yaml (hunk 3 above)
Comment (shaikatz) [reply pkg/controller/controller-gen.yaml:6]: It used everywhere in the project where you find `c.Kubernetes`

File: deploy/rbac.yaml (hunk 4 above)
Comment (shaikatz) [reply deploy/rbac.yaml]: Deleted

**Pull request merged**